### ISSUE TITLE
Imagenes wordpress (enhancement)

### ISSUE THREAD

**Issue by MarceloGallardoMadrid** [benjumea12/la-filmo#6]
DSR!(Usuario discord)
hay que configurar en wordpress cual es el "post-thumbnail" que se usa para las imagenes de los thumbnail sino se usa la mas grande
entiendo que la correcta seria las que estan escaladas en 1024x576
ese cambio haria que la pagina cargue mucho mas rapida
asi con el ojometro estimo que con ese cambio cargarian 4 veces mas rapido

**Comment by xchwarze** [comment]
Esto esta implementado ya en algunos lados, por lo menos lo vi aca https://github.com/benjumea12/la-filmo/commit/7f15bf682aeef3f8f2bb8fe0188cd38c860f1b98

**Comment by benjumea12** [comment]
Sí, está se está implementando en las paginas que contienen mas imágenes (excelente sugerencia)
